app/src/main/java/org/dhis2/data/forms/dataentry/fields/scan/ScanTextView.kt

```diff
@@ -1,57 +1,20 @@
 package org.dhis2.data.forms.dataentry.fields.scan
 
-import android.app.Activity
 import android.content.Context
-import android.content.Intent
-import android.text.Editable
-import android.text.TextUtils
-import android.text.TextWatcher
 import android.util.AttributeSet
-import android.view.MotionEvent
-import android.view.View
-import android.view.inputmethod.EditorInfo.IME_ACTION_DONE
-import android.view.inputmethod.EditorInfo.IME_ACTION_NEXT
-import android.widget.ImageView
-import android.widget.TextView
-import androidx.core.content.res.ResourcesCompat
 import androidx.databinding.ViewDataBinding
-import androidx.fragment.app.FragmentActivity
-import com.google.android.material.textfield.TextInputEditText
-import com.google.android.material.textfield.TextInputLayout
 import org.dhis2.BR
-import org.dhis2.Bindings.closeKeyboard
-import org.dhis2.R
-import org.dhis2.commons.dialogs.CustomDialog
-import org.dhis2.commons.resources.ColorUtils
 import org.dhis2.databinding.ScanTextViewAccentBinding
 import org.dhis2.databinding.ScanTextViewBinding
-import org.dhis2.usescases.qrScanner.ScanActivity
-import org.dhis2.usescases.searchTrackEntity.SearchTEActivity
-import org.dhis2.utils.ActivityResultObservable
-import org.dhis2.utils.ActivityResultObserver
-import org.dhis2.utils.Constants
-import org.dhis2.utils.Constants.EXTRA_DATA
-import org.dhis2.utils.Constants.RQ_QR_SCANNER
-import org.dhis2.utils.Preconditions.Companion.equals
 import org.dhis2.utils.customviews.FieldLayout
-import org.hisp.dhis.android.core.common.ValueTypeRenderingType
 
 class ScanTextView @JvmOverloads constructor(
```

Comment:
Is this file being using actually?. Could you remove it?

Reply:
Removed!

Reply:
👏🏽